<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeFileDialog</class>
 <widget class="QDialog" name="ChangeFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Data File</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select by source file</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="Kst::ComboBox" name="_files">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_allFromFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add All &amp;From</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string>Vector and Matrix Selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="_changeFilePrimitiveList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>The source file for the vectors selected in this list will be changed.</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="_addAll">
            <property name="whatsThis">
             <string>Select all vectors in the list.</string>
            </property>
            <property name="text">
             <string>Add A&amp;ll</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_add">
            <property name="whatsThis">
             <string>Select all vectors in the list.</string>
            </property>
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_remove">
            <property name="whatsThis">
             <string>Unselect all vectors in the list.</string>
            </property>
            <property name="text">
             <string>&amp;Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="_removeAll">
            <property name="whatsThis">
             <string>Unselect all vectors in the list.</string>
            </property>
            <property name="text">
             <string>R&amp;emove All</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>51</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="_selectedFilePrimitiveList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>The source file for the vectors selected in this list will be changed.</string>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroup2">
     <property name="title">
      <string>New Data File</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>&amp;File:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_dataFile</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="Kst::DataSourceSelector" name="_dataFile" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="_changeSelected">
          <property name="text">
           <string>C&amp;hange selected vectors and matrices</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="_duplicateDependents">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>D&amp;uplicate dependents</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="_duplicateSelected">
          <property name="text">
           <string>&amp;Duplicate selected vectors and matrices</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Kst::ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::DataSourceSelector</class>
   <extends>QWidget</extends>
   <header>datasourceselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_changeFilePrimitiveList</tabstop>
  <tabstop>_removeAll</tabstop>
  <tabstop>_addAll</tabstop>
  <tabstop>_dataFile</tabstop>
  <tabstop>_changeSelected</tabstop>
  <tabstop>_duplicateSelected</tabstop>
  <tabstop>_duplicateDependents</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
